src/pages/Home.jsx

```diff
@@ -1,7 +1,328 @@
+import { useState, useEffect } from 'react';
+import {
+  Search,
+  ArrowRightCircle,
+  MapPin,
+  CalendarDaysIcon,
+  Star,
+  Scissors,
+  UsersIcon,
+  ShoppingBag,
+} from 'lucide-react';
+import image1 from '../assets/saloonImage1.jpg';
+import image2 from '../assets/saloonImage2.jpg';
+import image3 from '../assets/saloonImage3.webp';
+import gasabo from '../assets/gasabo.jpg';
+import nyarugenge from '../assets/nyarugenge.png';
+import kicukiro from '../assets/kicukiroDistrict.webp';
+import testimonial from '../assets/testimonial.jpg';
+import SearchInputs from '../components/SearchInputs';
+
 const Home = () => {
+  const [isHoverd, setIsHoverd] = useState(false);
+  const [current, setCurrent] = useState(0);
+
+  const districtDetails = [
+    {
+      id: 1,
+      image: gasabo,
+      title: 'Gasabo District',
+      location: 'Kigali, Rwanda',
+      TatalSaloons: '45',
+    },
+    {
+      id: 2,
+      image: nyarugenge,
+      title: 'Nyarugenge District',
+      location: 'Kigali, Rwanda',
+      TatalSaloons: '45',
+    },
+    {
+      id: 3,
+      image: kicukiro,
+      title: 'Kicukiro District',
+      location: 'Kigali, Rwanda',
+      TatalSaloons: '45',
+    },
+  ];
+
+  const images = [image1, image2, image3];
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setCurrent((previous) => (previous + 1) % images.length);
+    }, 5000);
+
+    return () => clearInterval(interval);
+  }, [current]);
+
+  const commonlyQuestionAsked = [
+    {
+      Question: 'Do salons offer home service?',
+      Answer:
+        'Some salons do! Look for the Home Service Available tag on the salon’s profile',
+    },
+    {
+      Question: ' Can I cancel or reschedule my appointment?',
+      Answer:
+        'Yes! You can cancel or reschedule your appointment through your account before the scheduled time.',
+    },
+    {
+      Question: 'How do I find salons near me?',
+      Answer:
+        'Use our search feature to find salons by district or location and choose the one that fits your needs.',
+    },
+    {
+      Question: 'What types of beauty services are available?',
+      Answer:
+        'Our partnered salons offer haircuts, styling, coloring, manicures, pedicures, facials, and more!',
+    },
+  ];
```

Comment:
Move this in another file and import it to minimise the size of this file and make sure that it is reusable